File: app/src/main/java/br/com/jonathanarodr/playmovie/model/MovieVideo.java
package br.com.jonathanarodr.playmovie.model;

import com.google.gson.annotations.SerializedName;

public class MovieVideo {

    private static final String YOUTUBE_URL = "https://www.youtube.com/watch?v=";

    @SerializedName("key")
    private String key;
    @SerializedName("site")
    private String site;

    public String getKey() {
        return YOUTUBE_URL + key;
    }

    public String getSite() {
        return site;
    }

}

File: app/src/main/java/br/com/jonathanarodr/playmovie/model/MovieVideo.kt
package br.com.jonathanarodr.playmovie.model

import com.google.gson.annotations.SerializedName

data class MovieVideo(
    @SerializedName("key")
    val key: String,
    @SerializedName("site")
    val site: String
) {
    companion object {
        private const val YOUTUBE_URL = "https://www.youtube.com/watch?v="
    }

    fun getUrlVideo() = YOUTUBE_URL + key
}
